README.md

```diff
@@ -6,15 +6,57 @@
 
 ## Quickstart
 
+### CMSSW with EL8/EL9
+
+We use the [scram-venv](http://cms-sw.github.io/venv.html) utility to create a
+python virtual environment in our CMSSW area:
+
+```bash
+cmsrel CMSSW_14_1_0_pre4
+cd CMSSW_14_1_0_pre4/src
+scram-venv
```

Comment:
Tried running this - Should cmsenv should come before scram-venv?

Reply:
For me it worked in that order. But I guess from http://cms-sw.github.io/venv.html it should be
```
cmsenv
scram-venv
cmsenv
```
?